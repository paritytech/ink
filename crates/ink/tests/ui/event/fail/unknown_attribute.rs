#[ink::event]
#[ink(anonymous)]
#[ink(my_arg)]
pub struct Event {
    pub topic: [u8; 32],
}

fn main() {}
