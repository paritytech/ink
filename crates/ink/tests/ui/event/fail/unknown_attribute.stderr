error: encountered unknown or unsupported ink! event item configuration argument
 --> tests/ui/event/fail/unknown_attribute.rs:3:7
  |
3 | #[ink(my_arg)]
  |       ^^^^^^

error[E0277]: the trait bound `Event: ink::ink_env::Event` is not satisfied
 --> tests/ui/event/fail/unknown_attribute.rs:1:1
  |
1 | #[ink::event]
  | ^^^^^^^^^^^^^ the trait `ink::ink_env::Event` is not implemented for `Event`
  |
  = note: this error originates in the attribute macro `ink::event` (in Nightly builds, run with -Z macro-backtrace for more info)
